src/app/containers/manuscript/manuscript-editor.tsx

```diff
@@ -135,6 +136,7 @@ export const ManuscriptEditor: React.FC = () => {
       />
       {renderKeywords(allKeywords)}
       <ArticleInformation />
+      <RelatedArticles />
```

Comment:
I think this might need to be after the references, but I'm not 100%. Can either merge and alter later or run through it with the designer beforehand.